package.json

```diff
@@ -76,6 +77,8 @@
     ]
   },
   "devDependencies": {
+    "ava": "^5.3.1",
```

Comment:
can we reuse the existing test framework, instead?

Reply:
sorry I see your PR comment - can ignore for now as I don't want you to waste time porting tests